src/bp-core/classes/class-bp-email-tokens.php

```diff
@@ -1152,7 +1152,7 @@ public function token__group_invite_message( $bp_email, $formatted_tokens, $toke
 						<tr>
 							<td>
 								<div style="color: <?php echo esc_attr( $settings['body_text_color'] ); ?>; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; font-size: <?php echo esc_attr( $settings['body_text_size'] . 'px' ); ?>; letter-spacing: -0.24px; line-height: <?php echo esc_attr( floor( $settings['body_text_size'] * 1.625 ) . 'px' ); ?>;">
-									<?php echo wpautop( $formatted_tokens['invites.message'] ); ?>
+									<?php echo wpautop( $tokens['invite.message'] ); ?>
```

Comment:
@KartikSuthar why we changed the $formatted_tokens['invites.message'] to $tokens['invite.message'] ? Can you explain me more?

Reply:
@chetansatasiya, due to parameter change.
https://github.com/buddyboss/buddyboss-platform/pull/586/files#diff-d1844393616e1cb51618374627a1ff71R368